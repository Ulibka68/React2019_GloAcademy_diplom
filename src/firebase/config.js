
  const firebaseConfig = {
    apiKey: process.env.REACT_APP_API_KEY,
    authDomain: "vlasov-react-gloacademy-diplom.firebaseapp.com",
    databaseURL: "https://vlasov-react-gloacademy-diplom.firebaseio.com",
    projectId: "vlasov-react-gloacademy-diplom",
    storageBucket: "vlasov-react-gloacademy-diplom.appspot.com",
    messagingSenderId: process.env.REACT_APP_MESSAGINGSENDERID,
    appId: process.env.REACT_APP_APPID
  };




  export default firebaseConfig;