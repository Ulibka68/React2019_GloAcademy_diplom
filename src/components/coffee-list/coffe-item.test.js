import React from "react";
import {shallow} from "enzyme";
import CoffeeItem from "./coffe-item";
import config from "../../services/setupTests";
// import renderer from 'react-test-renderer';

const props = {
    url : "https://www.sciencenews.org/sites/default/files/main/articles/100315_coffee_opener_NEW_0.jpg",
    name : "Solimo Coffee Beans 2kg",
    country : 'Brazil',
    price : "10.73",
    id : "1"
};

describe('<CoffeeItem render />', () => {
    it('renders correctly', () => {
        const tree = shallow(<CoffeeItem {...props} />);
          
        // expect(tree).toMatchSnapshot();
        expect(tree.debug()).toMatchSnapshot();
    
        console.log(tree);
        // expect (tree.prop('name')).toEqual("Solimo Coffee Beans 2kg");
      });

      // видео 27:44
      // it('props CoffeeItem', ()=> {
      //   const tree = shallow(<CoffeeItem {...props} />);
      //   expect(tree.prop('id')).toEqual("1");
      // });
    });  
    


/*

describe('<CoffeeItem render />', () => {
it('renders correctly', () => {
    const tree = renderer
      .create(<CoffeeItem 
        url = "https://www.sciencenews.org/sites/default/files/main/articles/100315_coffee_opener_NEW_0.jpg"
            name = "Solimo Coffee Beans 2kg"
            country = 'Brazil'
            price = "10.73"
            id = "1" />)
      .toJSON();
    expect(tree).toMatchSnapshot();

    console.log(tree);
    // expect (tree.prop('name')).toEqual("Solimo Coffee Beans 2kg");
  });
});  

describe('Test <CoffeeItem>', () => {

    
    
    // тест на props
    it('PROPS test', () => {
        const coffeeItem  =shallow(<CoffeeItem 
            
            name = "Solimo Coffee Beans 2kg"
            country = 'Brazil'
            price = "10.73"
            id = "1"
        />);

        
        expect(coffeeItem.props().country).to.equal('Brazil!');
        });
    
      
    });

*/