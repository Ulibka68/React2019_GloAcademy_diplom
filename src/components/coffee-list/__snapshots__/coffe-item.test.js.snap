// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<CoffeeItem render /> renders correctly 1`] = `
"<div className=\\"shop__item\\">
  <img src=\\"https://www.sciencenews.org/sites/default/files/main/articles/100315_coffee_opener_NEW_0.jpg\\" style={{...}} alt=\\"coffee\\" onClick={[Function: onClick]} />
  <div className=\\"shop__item-title\\">
    Solimo Coffee Beans 2kg
  </div>
  <div className=\\"shop__item-country\\">
    Brazil
  </div>
  <div className=\\"shop__item-price\\">
    10.73
    $
  </div>
</div>"
`;
